catalog/spatial/registry/pom.xml

```diff
@@ -157,6 +157,6 @@
         <module>registry-federation-admin-impl</module>
         <module>registry-app</module>
         <module>registry-admin-modules</module>
-
+        <module>registry-report-viewer</module>
```

Comment:
It would be nice if we could put them both in the same module since you wouldn't want to have the action provider running without the viewer. Let's talk about it tomorrow.